src/funcs.py

```diff
@@ -26,18 +45,41 @@ def longest(x, y):
     Returning the longest of two lists.
 
     >>> longest([1, 2, 3], [4, 5])
-    'TEST ME'
+    [1, 2, 3]
+
+    >>> longest([1,2], [3,4])
+    
     """
+    if len(x)==len(y):
+        return None 
```

Comment:
One could argue that the longest list, if they are the same length, would be any of them, rather than `None`. But we didn't specify that, so this is fine.